cookbooks/java/metadata.rb

```diff
@@ -45,9 +45,9 @@
 end
 
 depends 'apt'
-depends 'windows'
-depends 'homebrew'
+# depends 'windows'
```

Comment:
Can you leave a comment somewhere about removing these lines, so that the next person to update the java recipe knows where to look for additional changes? It could be a "README.java" or similar that lives in the same commit and says "run git log on this file" to see the changes.